Auto instrumentation updates v2 (#28766)
DataDog/documentation: content/en/real_user_monitoring/browser/setup/server/_index.md

<div class="alert alert-warning">To request support for a web server that is not listed here, <a href="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/">fill out this form.</a></div>

> If you don't want to manage your web servers remotely, follow the documentation for your web server type to set up your RUM application:

Reviewer: **suggestion:** Wording is a bit weird here. Customers are not "managing web servers remotely", they are managing the RUM configuration remotely.

```suggestion
If you don't want to configure your RUM applications remotely, follow the documentation for your web server type to configure your RUM application locally:
```

<br>
{{< partial name="rum/rum-browser-setup.html" >}}
<br>

## Limitations

Keep in mind the following limitations when using this setup:

- This instrumentation method **does not support [advanced RUM configurations][1]**, except for `allowedTracingUrls` and `excludedActivityUrls`.

> - If your web server is acting as a proxy and the upstream server uses **end-to-end encryption (TLS)** or **content compression** (gzip, zstd, Brotli), the RUM Browser SDK may **not be injected**. To ensure proper instrumentation:

Reviewer 2: Not clear if this means "the SDK _cannot_ be injected" or "in some cases, the SDK _might not_ be injected." My guess is the former:
```suggestion
- If your web server is acting as a proxy and the upstream server uses **end-to-end encryption (TLS)** or **content compression** (gzip, zstd, Brotli), the RUM Browser SDK **cannot be injected**. To ensure proper instrumentation:
```
